test/widgets/message_list_test.dart

```diff
@@ -605,6 +606,86 @@ void main() {
           expectedMessage: 'NetworkException: Oops (ClientException: Oops)');
       });
     });
+
+    group('Update Narrow on message move', () {
+      final channel = eg.stream(name: 'move test stream');
+      final message = eg.streamMessage(stream: channel, content: 'Message to move');
```

Comment:
The message gets mutated. So each test should make it fresh — else it gets the version mutated by the last test.